lib/plugins/inspec-compliance/lib/inspec-compliance/cli.rb

```diff
@@ -6,8 +6,7 @@ module InspecPlugins
   module Compliance
     class CLI < Inspec.plugin(2, :cli_command)
       include Inspec::Dist
-
-      subcommand_desc "compliance SUBCOMMAND", "#{COMPLIANCE_PRODUCT_NAME} commands"
+      subcommand_desc "automate SUBCOMMAND or compliance SUBCOMMAND", "#{COMPLIANCE_PRODUCT_NAME} commands"
 
       # desc "login https://SERVER --insecure --user='USER' --ent='ENTERPRISE' --token='TOKEN'", 'Log in to a Chef Compliance/Chef Automate SERVER'
       desc "login", "Log in to a #{COMPLIANCE_PRODUCT_NAME}/#{AUTOMATE_PRODUCT_NAME} SERVER"
```

Comment:
When you run `inspec automate help`, you see the word `Compliance Server` a lot. I think that is fairly nonsensical now.  Those occurences should be replaced with AUTOMATE_PRODUCT_NAME.

Here, just eliminate the alternative:
`Log in to a #{AUTOMATE_PRODUCT_NAME} SERVER`

Reply:
But in each of the command `desc`, let's replace COMPLIANCE_PRODUCT_NAME with AUTOMATE_PRODUCT_NAME.